src/inc-js-variants.php

```diff
@@ -239,14 +180,14 @@ className: '',
         $.get('ajax/convert_position.php', { variant: sVariantNotation, gene: sGene },
             function(sData) {
                 if (sData != '<?php echo AJAX_DATA_ERROR; ?>' && sData != '<?php echo AJAX_FALSE; ?>' && sData != '<?php echo AJAX_NO_AUTH; ?>') {
-                    if (oThisDNA.attr('name') == 'VariantOnGenome/DNA') {
+                    if (oThisDNA.filter("[name^='VariantOnGenome/DNA']").size()) {
                         // This function was called from the genomic variant, so fill in the return values from mutalyzer in the transcript DNA fields.
                         aVariants = sData.split(';');
                         var nVariants = aVariants.length;
                         for (i = 0; i < nVariants; i++) {
                             var aVariant = /^([A-Z]{2}_\d{6,9}\.\d{1,2}(?:\([A-Z0-9]+_v\d{3}\))?):([cn]\..+)$/.exec(aVariants[i]);
                             if (aVariant != null) {
-                                var oInput = $('#variantForm input[id_ncbi="' + aVariant[1] + '"]');
+                                var oInput = $('#variantForm input[data-id_ncbi="' + aVariant[1] + '"]');
```

Comment:
Although this notation is easier, I think there's maybe a reason why the filters on the data fields normally work differently?